### PR TITLE
update 2.x azure storage blob trigger documentation

### PR DESCRIPTION
**Pull request by joachimgoris**
Signed-off-by: Joachim Goris <gorisjoachim@hotmail.com>

#510 

Add `accountName` in Azure Storage Blob trigger documentation.

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

### CODE REVIEW

File: content/docs/2.0/scalers/azure-storage-blob.md [hunk 1]
```diff
@@ -37,7 +39,11 @@ You can authenticate by using pod identity or connection string authentication.
 
 **Connection String Authentication:**
 
-- `connection` - Connection string for Azure Storage Account
+- `connection` - Connection string for Azure Storage Account.
+
+**PodIdentity Authentication**
```
Comment (tomkerkhove) [inline content/docs/2.0/scalers/azure-storage-blob.md:44]: Nit, can you update the files to use this formatting, please?
```suggestion
**Pod Identity Authentication**
```

**Review by tomkerkhove — APPROVED** [review]
LGTM, just one request.

Thank you!

File: content/docs/2.0/scalers/azure-storage-blob.md (hunk 1 above)
Comment (joachimgoris) [reply content/docs/2.0/scalers/azure-storage-blob.md]: I split 'PodIdentity' to 'Pod Identity'

**Comment by tomkerkhove** [comment]
Bedankt!

**Pull request merged**